docs/multidispatching.md

```diff
@@ -1,29 +1,29 @@
 # Multi-Dispatching
 
-Multidispatching allows you to define methods and functions which should
-behave differently based on arguments' types without cluttering
+Multidispatching allows Gaphor to define methods and functions which should
+behave differently based on the argument types without cluttering
 `if-elif-else` chains and `isinstance` calls.
 
-All you need is inside `generic.multidispatch` module. See examples
+All you need is inside `gaphor.utils.generic.multidispatch` module. See examples
 below to learn how to use it to define multifunctions and multimethods.
 
 ## Multifunctions
 
 Suppose we want to define a function which behaves differently based on
-arguments' types. The naive solution is to inspect argument types with
+arguments' types. One solution may be to inspect argument types with
 `isinstance` function calls but generic provides us with
 `@multidispatch` decorator which can easily reduce the amount of
 boilerplate and provide desired level of extensibility:
 
-    from generic.multidispatching import multidispatch
+    from gaphor.utils.generic.multidispatching import multidispatch
 
     @multidispatch(Dog)
     def sound(o):
-      print "Woof!"
+      print("Woof!")
 
     @sound.when(Cat)
```

Comment:
`when` should be `register`, as is with single-dispatch.

Reply:
I think we can remove this page now that we have are just using the generic module. This probably needs to be updated in the generic docs.